packages/evo-marko/src/tags/evo-eek/index.marko

```diff
@@ -0,0 +1,89 @@
+import type { AttrString } from "marko/tags-html";
+export interface Input extends Marko.HTML.Div {
+  max: string;
+  min: string;
+  rating: string;
+  /**
+   *  The aria-label accessibility label for the eek rating. This is for internationalization.
+   *  It should use eekRating, min, and max values in the label in order to demonstrate
+   *  to screen readers the content on the component.
+   *  Expected return `Energy Rating: ${eekRating}. Range: ${max} - ${min}.`
+   **/
```

Comment:
Again, this is probably intended to be multiple paragraphs which should be separated by a blank line